guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -58,6 +58,15 @@ ifdef::katello[]
 # {package-manager} config-manager --set-enabled powertools
 ----
 endif::[]
+ifdef::foreman-el,katello[]
```

Comment:
I'm not sure if this needs a guard on katello. IMHO this is valid for vanilla Foreman on EL8 as well.

Reply:
That's not a guard, but rather a "which guides to include this in" line. So in this case, include it in the Foreman Enterprise Linux guide and the Katello guide.

Reply:
So I read `ifdef` as an AND, not an OR. IMHO this should be an OR. If it is, then consider this addressed.